dch/cog.py

```diff
@@ -67,7 +67,7 @@ async def send_bot_help(self, mapping):
 
                 embed.add_field(name=cog.qualified_name, value=cog_help)
 
-        embed.set_thumbnail(url=ctx.bot.user.display_avatar.url)
```

Comment:
You might be using discord.py 1.7.3 but this module is made for >= v2.0 and according to https://github.com/Rapptz/discord.py/blob/master/discord/member.py#L583 this discord.py supports display_avatar.url